src/OpenTelemetry.Instrumentation.Runtime/README.md

```diff
@@ -60,6 +60,138 @@ using var meterProvider = Sdk.CreateMeterProviderBuilder()
+| process.runtime.dotnet.**thread_pool.queue.length**          | The number of work items that are currently queued to be processed by the thread pool.                                                                                                                                                                                              | `{items}`   | ObservableGauge   | `Int64`    |                  |                  |
+| process.runtime.dotnet.**timer.count**                       | The number of timer instances that are currently active. Timers can be created by many sources such as System.Threading.Timer, Task.Delay, or the timeout in a CancellationSource. An active timer is registered to tick at some point in the future and has not yet been canceled. | `{timers}`  | ObservableGauge   | `Int64`    |                  |                  |
+
+The APIs used to retrieve the values are:
+
+* [Monitor.LockContentionCount](https://docs.microsoft.com/dotnet/api/system.threading.monitor.lockcontentioncount?view=netcore-3.1):
+  Gets the number of times there was contention when trying to take the monitor's
+  lock.
+* [ThreadPool.ThreadCount](https://docs.microsoft.com/dotnet/api/system.threading.threadpool.threadcount?view=netcore-3.1):
+  Gets the number of thread pool threads that currently exist.
+* [ThreadPool.CompletedWorkItemCount](https://docs.microsoft.com/dotnet/api/system.threading.threadpool.completedworkitemcount?view=netcore-3.1):
+  Gets the number of work items that have been processed so far.
+* [ThreadPool.PendingWorkItemCount](https://docs.microsoft.com/dotnet/api/system.threading.threadpool.pendingworkitemcount?view=netcore-3.1):
+  Gets the number of work items that are currently queued to be processed.
+* [Timer.ActiveCount](https://docs.microsoft.com/dotnet/api/system.threading.timer.activecount?view=netcore-3.1):
+  Gets the number of timers that are currently active. An active timer is registered
+  to tick at some point in the future, and has not yet been canceled.
+
+### Assemblies related metrics
+
+| Name                                      | Description                                              | Units          | Instrument Type | Value Type | Attribute Key(s) | Attribute Values |
+|-------------------------------------------|----------------------------------------------------------|----------------|-----------------|------------|------------------|------------------|
+| process.runtime.dotnet.**assembly.count** | The number of .NET assemblies that are currently loaded. | `{assemblies}` | ObservableGauge | `Int64`    |                  |                  |
```

Comment:
```suggestion
| process.runtime.dotnet.**assemblies.count** | The number of .NET assemblies that are currently loaded. | `{assemblies}` | ObservableGauge | `Int64`    |                  |                  |
```

I think the pattern elsewhere is that these counts are pluralized?

Reply:
Indeed! As is recommended by the semantic convention: https://github.com/open-telemetry/opentelemetry-specification/tree/main/specification/metrics/semantic_conventions#pluralization